### PR TITLE
Added reg 0.10

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

**Comment by stuertz** [comment]
@conda-forge/core It is ready for merge.

File: recipes/reg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - repoze.lru
+
+test:
+  imports:
+    - reg
+
+about:
+  home: http://reg.readthedocs.io
+  license: BSD
```
Comment (jakirkham) [inline recipes/reg/meta.yaml]: What version (e.g. `BSD 3-Clause` for the 3-Clause license).

File: recipes/reg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - repoze.lru
```
Comment (jakirkham) [inline recipes/reg/meta.yaml]: Is `repoze.lru` needed at build time or only install time? If the latter, please drop this.

**Comment by stuertz** [comment]
@jakirkham Could you please restart the circleci build? It seemed to be successful, but but didn't update the state here.

**Comment by ocefpaf** [comment]
@stuertz CircleCI did finish here I guess that only the status was not updated. See https://circleci.com/gh/conda-forge/staged-recipes/13900?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Pull request merged**